management/server/http/handler.go

```diff
@@ -113,6 +113,7 @@ func (apiHandler *apiHandler) addUsersEndpoint() {
 	apiHandler.Router.HandleFunc("/users/{userId}", userHandler.UpdateUser).Methods("PUT", "OPTIONS")
 	apiHandler.Router.HandleFunc("/users/{userId}", userHandler.DeleteUser).Methods("DELETE", "OPTIONS")
 	apiHandler.Router.HandleFunc("/users", userHandler.CreateUser).Methods("POST", "OPTIONS")
+	apiHandler.Router.HandleFunc("/users/{userId}/invite", userHandler.InviteUser).Methods("POST", "OPTIONS")
```

Comment:
Why do we need userID in the path? as far as I can see it is not used so /users/invite should be fine

Reply:
We just had a talk and it might make more sense to drop email and go with userID directly?

Reply:
> Why do we need userID in the path? as far as I can see it is not used so /users/invite should be fine

The userID is used in setting initiator ID when storing the event